docs/content/integrations/airlift/tutorial/migrate.mdx

```diff
@@ -0,0 +1,243 @@
+# Airflow Migration Tutorial: Migrating Assets
+
+Previously, we completed the ["Observe" stage](/integrations/airlift/tutorial/observe) of the Airflow migration process by encoding the assets that are produced by each task. We also introduced partitioning to those assets.
+
+Up until now, we have not touched the Airflow code base at all. Now it's time to begin the actual migration process.
+
+Once you have created corresponding definitions in Dagster to your Airflow tasks, you can proxy execution to Dagster on a per-task basis while Airflow is still controlling scheduling and orchestration. Once a task has been proxied, Airflow will kick off materializations of corresponding Dagster assets in place of executing the business logic of that task.
```

Comment:
```suggestion
Once you have created the corresponding definitions in Dagster for your Airflow tasks, you can proxy execution to Dagster on a per-task basis while Airflow still controls scheduling and orchestration. Once a task has been proxied, Airflow will kick off materializations of corresponding Dagster assets instead of executing the business logic of that task.
```